libraries/botbuilder-dialogs-declarative/package.json

```diff
@@ -28,20 +28,22 @@
     }
   },
   "dependencies": {
+    "bf-cli": "^1.0.4",
     "botbuilder-core": "4.1.6",
     "botbuilder-dialogs": "4.1.6",
     "botbuilder-stdlib": "4.1.6",
     "chokidar": "^3.4.0"
   },
   "devDependencies": {
+    "jsonschema": "^1.4.0"
   },
   "scripts": {
     "build": "tsc -b",
     "build-docs": "typedoc --theme markdown --entryPoint botbuilder-dialogs-adaptive --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\botbuilder-dialogs .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Dialogs\" --readme none",
     "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
     "lint": "eslint . --ext .js,.ts",
     "postbuild": "downlevel-dts lib _ts3.4/lib --checksum",
-    "test": "yarn build && nyc mocha tests/ --exit",
+    "test": "yarn build && nyc mocha tests/**/*.test.js --exit",
```

Comment:
I would leave the path as `tests/` and add `--recursive` instead, that's the approach we've been taking in other packages.

Reply:
Here, I'm just trying to separate out running the normal tests and the schema tests. I did so by having the schema test *not* named `*.test.js`. Would it be better to move to a separate directory?